pre-docs/applying_manifest_blocks.md

```diff
@@ -8,6 +8,8 @@ Within a plan, you can use Bolt to apply blocks of Puppet code (manifest blocks)
 
 [Writing plans](writing_plans.md)
 
+**Note:** If you installed bolt as a ruby gem, you'll need to `cd` into the bolt directory and run `bolt puppetfile install` to install required puppet modules
```

Comment:
I don't think this actually works, since `bolt puppetfile install` assumes the Puppetfile is inside `Boltdir`... You would need to just regular `r10k puppetfile install`.